ravedude/src/main.rs

```diff
@@ -2,6 +2,9 @@ use anyhow::Context as _;
 use colored::Colorize as _;
 use structopt::clap::AppSettings;
 
+#[macro_use]
+extern crate anyhow;
+
```

Comment:
Please drop this, `macro_use` and `extern crate` aren't really ideomatic for 2018 rust anymore.  You'll then need to refer to the macro as `anyhow::anyhow!()`, I've left comments above.